dpppt-config-backend/src/main/java/org/dpppt/switzerland/backend/sdk/config/ws/controller/DPPPTConfigController.java

```diff
@@ -78,11 +89,10 @@ public DPPPTConfigController() {
 		return ResponseEntity.ok().cacheControl(CacheControl.maxAge(Duration.ofMinutes(5))).body(config);
 	}
 
+	@Documentation(description = "Infobox testing endpoint", responses = {"200 => Infobox in all languages"})
 	@CrossOrigin(origins = { "https://editor.swagger.io" })
 	@GetMapping(value = "/testinfobox/config")
-	public @ResponseBody ResponseEntity<ConfigResponse> getGhettoboxConfig(
-			@RequestParam(required = true) String appversion, @RequestParam(required = true) String osversion,
-			@RequestParam(required = true) String buildnr) {
+	public @ResponseBody ResponseEntity<ConfigResponse> getGhettoboxConfig() {
```

Comment:
same here, we may want to test for different versions, and if it behaves differently (during later client tests). Always changing the API-Endpoint seems like a bad idea.

Reply:
added all arguments to both endpoints again.